Xamarin.Forms.Platform.Android/CarouselView/CarouselViewRenderer.cs

```diff
@@ -312,9 +324,9 @@ void ViewPager_PageSelected(object sender, ViewPager.PageSelectedEventArgs e)
 		// To invoke PositionSelected
 		void ViewPager_PageScrollStateChanged(object sender, ViewPager.PageScrollStateChangedEventArgs e)
 		{
-			// ScrollStateIdle = 0 : the pager is in Idle, settled state
+			// ScrollStateIdle = 0 : the pager is in Idle, Updatetled state
```

Comment:
I think a replace-all (Set -> Update) went incorrect here and two lines below...